packages/waku/src/lib/constants.ts

```diff
@@ -1,3 +1,4 @@
 export const EXTENSIONS = ['.js', '.ts', '.tsx', '.jsx', '.mjs', '.cjs'];
 export const SRC_MAIN = 'main';
 export const SRC_ENTRIES = 'entries';
+export const FUNCTION_RESULT = Symbol.for('waku.function.result');
```

Comment:
It seems that

```
export const FUNCTION_RESULT = '_value';
```

would accomplish a similar goal while not introducing `Symbol.for`?

Reply:
I'm fine with that, if that helps a lot.


btw, consider `_` prefix means "reserved" in this case.

We used to have this check:
https://github.com/dai-shi/waku/blob/8a54eab1cb3bb04296f57c77aae999fda328184f/packages/waku/src/lib/renderers/rsc-renderer.ts#L139-L141